README.md

```diff
@@ -72,21 +72,41 @@ jobs:
     # the ${{ steps.buildnumber.outputs.build_number }} output.
 ```
 
+## Generating multiple independent build numbers
+
+Sometimes you may have more than one project to build in one repository. For example you may have a client and a server in the same github repository that you would like to generate independent build numbers for. Another example is you have two Dockerfiles in one repo and you'd like to version each of the built images with their own numbers.  
+To do this, use the `prefix` key, like so:
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Generate build number
+      id: buildnumber
+      uses: einaregilsson/build-number@v1 
+      with:
+        token: ${{ secrets.github_token }}
+        prefix: client
+```
+
+This will generate a git tag like `client-build-number-1`.
+
+If you then do the same in another workflow and use `prefix: server` then you'll get a second build-number tag called `server-build-number-1`.
 
 ## Setting the initial build number.
 
-If you're moving from another build system, you might want to start from some specific number. The ```build-number``` action simply uses a special tag name to store the build number, ```build-number-x```, so you can just create and push a tag with the number you want to start on. E.g. do
+If you're moving from another build system, you might want to start from some specific number. The `build-number` action simply uses a special tag name to store the build number, `build-number-x`, so you can just create and push a tag with the number you want to start on. E.g. do
 
 ```
 git tag build-number-500
 git push origin build-number-500
 ```
 
-and then your next build number will be 501. The action will always delete older refs that start with ```build-number-```, e.g. when it runs and finds ```build-number-500``` it will create a new tag, ```build-number-501``` and then delete ```build-number-500```.
+and then your next build number will be 501. The action will always delete older refs that start with `build-number-`, e.g. when it runs and finds `build-number-500` it will create a new tag, `build-number-501` and then delete `build-number-500`.
 
 ## Branches and build numbers
 
 The build number generator is global, there's no concept of special build numbers for special branches, it's probably something you would just use on builds from your master branch. It's just one number that gets increased every time the action is run.
```

Comment:
Maybe add something in here about prefixes, i.e. 

The build number generator is global, there's no concept of special build numbers for special branches (unless you handle it manually yourself, using prefixes) ...

Reply:
Let me know if my wording makes sense.